tasks/managed/add-fbc-contribution/add-fbc-contribution.yaml

```diff
@@ -238,6 +238,7 @@ spec:
         for((i=0; i<LENGTH; i++)); do
           ocp_version=$(jq -cr --argjson i "$i" '.components[$i].ocpVersion' "${SNAPSHOT_PATH}")
```

Comment:
Do you plan to use this for anything or it is just used to report the version in the result artifact?

Reply:
The version of it is set in the `prepare-fbc-release`. This is used later by the `publish-fbc-release` to determine if it should do just a single publishing or several.